demisto_sdk/__main__.py

```diff
@@ -438,7 +439,8 @@ def validate(config, **kwargs):
 @click.option('-a', '--artifacts_path', help='Destination directory to create the artifacts.',
               type=click.Path(file_okay=False, resolve_path=True), required=True)
 @click.option('--zip/--no-zip', help='Zip content artifacts folders', default=True)
-@click.option('--packs', help='Create only content_packs artifacts.', is_flag=True)
+@click.option('--packs', help='Create only content_packs artifacts. '
+                              'Used for server version 5.5.0 and lower.', is_flag=True)
```

Comment:
not sure i understand this flag, why is it used only for 5.5 and earlier ?

Reply:
This creates the conten_packs zip file which is used for versions 5.5. and lower